extensions/recyclerview/src/main/java/com/twocoders/extensions/recyclerview/RecyclerViewExtensions.kt

```diff
@@ -0,0 +1,18 @@
+package com.twocoders.extensions.recyclerview
+
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.StaggeredGridLayoutManager
+import com.twocoders.extensions.common.logd
+
+@BindingAdapter(value = ["spanCount"])
+fun RecyclerView.setSpanCount(spanCount: Int) {
+    with(layoutManager) {
```

Comment:
skor by sa mi tu hodilo `apply`, ale v konecnom dosledku je to uplne jedno...
```suggestion
    layoutManager.apply {
```